docs/guides/q-ctrl-performance-management.ipynb

```diff
@@ -176,325 +115,451 @@
+    "\n",
+    "| Name       | Type            | Description | Required | Default | Example |\n",
+    "|------------|---------------|------------------|----------|----------|--------------------|\n",
+    "| pubs       | `SamplerPubLike` or `list[SamplerPubLike]`   | One or more tuples containing the inputs listed under `SamplerPubLike` components     | Yes      | N/A | `(circuit, parameter_values)`            |\n",
+    "| instance   | `str`     | The hub/group/project to use in that format        | No | A Premium access instance is randomly chosen if your account has access to multiple instances     | `\"hub1/group1/project1\"`   |\n",
+    "| backend_name| `str`  | The name of the backend | No  |  The least busy backend that your instance has access to | `\"ibm_fez\"` |\n",
+    "| options| `dict`  | Input options; see Options section for more details | No  |  See the Options section for details | `{\"default_shots\": 2048}` |\n",
+    "\n",
+    "**`SamplerPubLike` components:**\n",
+    "- A single circuit defined as a `QuantumCircuit` or in OpenQASM 2.0 or 3.0 string format.\n",
+    "- (Optional) A collection of parameter values to bind the circuit against.\n",
+    "- (Optional) A dictionary of runtime options, such as the shot count.\n",
+    "\n",
+    "**Supported backends:**\n",
+    "The following list of backends are currently supported. If your device is not listed, [reach out to Q-CTRL](https://form.typeform.com/to/iuujEAEI?typeform-source=q-ctrl.com) to add support.\n",
+    "- ibm_brisbane  \n",
+    "- ibm_brussels  \n",
+    "- ibm_cleveland  \n",
+    "- ibm_fez  \n",
+    "- ibm_kawasaki  \n",
+    "- ibm_kyiv  \n",
+    "- ibm_nazca  \n",
+    "- ibm_quebec  \n",
+    "- ibm_rensselaer  \n",
+    "- ibm_sherbrooke  \n",
+    "- ibm_strasbourg  \n",
+    "- ibm_torino\n",
+    "\n",
+    "**Options:**\n",
+    "| Name   | Type   | Description  | Default |\n",
+    "|--------|----------|-----------------------|---------------------|\n",
+    "| session_id | `str`  | An existing Qiskit Runtime session ID  | `\"cw4r3je6f0t010870y3g\"` |\n",
+    "| default_shots | `int`  | The number of shots to use for each circuit  | `2048` |\n",
+    "\n",
+    "### Sampler outputs\n",
+    "| Name   | Type                                   | Description                                                                               | Example                         |\n",
+    "|--------|----------------------------------------|-------------------------------------------------------------------------------------------|---------------------------------|\n",
+    "| result | `list[ArrayLike[dict[str, int]]]`      | The resultant list of counts dictionaries corresponding to the list of input PUBs     | `[{'000001': 100, '000011': 2}]` |\n"
```

Comment:
Is the Sampler result not a `PrimitiveResult` ?

Reply:
Ah yes, it should be.